packages/tests/integration/migrations/1580658856631_author.ts

```diff
@@ -291,22 +293,6 @@ export function up(b: MigrationBuilder): void {
     json: { type: "jsonb", notNull: false },
   });
 
-  // for testing ignore of m2m
-  createManyToManyTable(b, "critics_to_tags", "critics", "tags");
```

Comment:
Moved the m2m tables below the `tasks` table so we can add a m2m to tasks (its only of the comment parent polys).